lib/storage.js

```diff
@@ -58,7 +58,7 @@ class Storage {
     let which = [true, false];
     if (_which === "staff") {
       which = [true];
-    } else if (_which === "contributors") {
+    } else if (_which === "volunteers") {
```

Comment:
```suggestion
    } else if (_which === "volunteers" || _which === "contributors") {
```
for easy roll out.

Reply:
Good idea. Do you plan to remove `|| _which === "contributors"` soon after launch? If so, I'll add an in-line comment about that along with a GitHub issue.